src/javascript/app/base/binary_loader.js

```diff
@@ -149,7 +149,8 @@ const BinaryLoader = (() => {
             return;
         }
 
-        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] });
+        const title = content.getElementsByTagName('h1')[0];
```

Comment:
please align the `=` with below

Reply:
I removed the title 😅 I guess it looks better without